### PR TITLE
Fix regex to detect ad type

### PR DESCRIPTION
**Pull request by zhouyx**
This is really tricky. 

Found it from #20534, they used \n to separate `amp-ad` and `width` instead of space. Fix the regex

### CODE REVIEW

File: build-system/app.js [hunk 1]
```diff
@@ -891,8 +868,9 @@ function escapeRegExp(string) {
 
 function elementExtractor(tagName, type) {
   type = escapeRegExp(type);
+  console.log('escapte type is /%s/', type);
   return new RegExp(
-      `<${tagName} [^>]*['"]${type}['"][^>]*>([\\s\\S]+?)</${tagName}>`,
+      `<${tagName}[(\\s)][^>]*['"]${type}['"][^>]*>([\\s\\S]+?)</${tagName}>`,
```
Comment (jridgewell) [inline build-system/app.js]: Delete the `(` and `)`

File: build-system/app.js [hunk 2]
```diff
@@ -33,15 +33,9 @@ const request = require('request');
 const pc = process;
 const countries = require('../examples/countries.json');
 const runVideoTestBench = require('./app-video-testbench');
-const {
```
Comment (jridgewell) [inline build-system/app.js]: Why so many unrelated changes?

Comment (zhouyx) [reply build-system/app.js]: oops, fixed

File: build-system/app.js (hunk 1 above)
Comment (zhouyx) [reply build-system/app.js]: done

**Comment by zhouyx** [comment]
@estherkim Please let me know when the visual diff build is fixed and that I can restart the build.  Thank you

**Comment by estherkim** [comment]
@zhouyx It's fixed! You should be good to go :)

**Comment by zhouyx** [comment]
Woohoo! Thank you for the quick fix!
